src/nvim/message.c

```diff
@@ -595,6 +636,28 @@ bool emsgf(const char *const fmt, ...)
   return ret;
 }
 
+#define MULTILINE_BUFSIZE 8192
+
+bool emsgf_echo(const char *const fmt, ...)
```

Comment:
`emsgf_pretty`

Reply:
what about "multiline" in general instead of "pretty" ?

Reply:
Sure. Or maybe `expanded`. Whatever you like best